# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mp4/MP4Metadata.cpp
# Commit: 89c241451a32
# Full Hash: 89c241451a32d362967315a0e2424fcce824208b
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-05-18 03:51:42
# Regressor Bug: 1817997
# File Overlap Count: 3
# Description:
#   Bug 1817997 - Update the MP4 demuxer to use TimeUnit based on the internal MP4 time base. r=alwu
#   
#   This still gets the initial time value from mp4parse-rust, that is in
#   microseconds. mp4parse-rust has been updated to expose real time, and will be
#   updated later.
# ==============================================================================

diff -r 66e4a26e1ebe -r 89c241451a32 dom/media/mp4/MP4Metadata.cpp
--- a/dom/media/mp4/MP4Metadata.cpp	Wed May 17 15:47:04 2023 +0000
+++ b/dom/media/mp4/MP4Metadata.cpp	Wed May 17 15:47:04 2023 +0000
@@ -24,10 +24,10 @@
 namespace mozilla {
 LazyLogModule gMP4MetadataLog("MP4Metadata");
 
-IndiceWrapper::IndiceWrapper(Mp4parseByteData& aIndice) {
+IndiceWrapper::IndiceWrapper(Mp4parseByteData& aRustIndice) {
   mIndice.data = nullptr;
-  mIndice.length = aIndice.length;
-  mIndice.indices = aIndice.indices;
+  mIndice.length = aRustIndice.length;
+  mIndice.indices = aRustIndice.indices;
 }
 
 size_t IndiceWrapper::Length() const { return mIndice.length; }